File: dev/requirements/screen/cases/countdown-warning.2.6.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 2.6 — the mockup's moment: 1:18 left, the countdown orange.
let countdownWarning_2_6 = RequirementCase(id: "2.6") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state(elapsed: 1122)), slug: "countdown-warning", id: "2.6", region: .clocks)
    }
}
#endif

File: dev/requirements/screen/cases/countdown-warning.2.6.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 2.6 — Inside the last three minutes the countdown turns orange.
final countdownWarning_2_6 = RequirementCase('2.6', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(elapsed: const Duration(minutes: 18, seconds: 42)),
    region: StageRegion.clocks,
    slug: 'countdown-warning',
    id: '2.6',
  );
});
